jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/ImageNameTest.java

```diff
@@ -179,42 +154,76 @@ public void namesUsedByDockerTests() {
     }
 
     @Test
-    public void testImageNameWithUsernameHavingPeriods() {
+    void testImageNameWithUsernameHavingPeriods() {
         // Given
         String name = "roman.gordill/customer-service-cache:latest";
 
         // When
         ImageName imageName = new ImageName(name);
 
         // Then
-        assertNotNull(imageName);
-        assertEquals("roman.gordill", imageName.getUser());
-        assertEquals("roman.gordill/customer-service-cache", imageName.getRepository());
-        assertEquals("latest", imageName.getTag());
-        assertNull(imageName.getRegistry());
+        assertThat(imageName).isNotNull();
+        assertThat(imageName.getUser()).isEqualTo("roman.gordill");
+        assertThat(imageName.getRepository()).isEqualTo("roman.gordill/customer-service-cache");
+        assertThat(imageName.getTag()).isEqualTo("latest");
+        assertThat(imageName.getRegistry()).isNull();
     }
 
     @Test
-    public void testImageNameWithNameContainingRegistryAndName() {
+    void testImageNameWithNameContainingRegistryAndName() {
         // Given
         String name = "foo.com:5000/customer-service-cache:latest";
 
         // When
         ImageName imageName = new ImageName(name);
 
         // Then
-        assertNotNull(imageName);
-        assertNull(imageName.getUser());
-        assertEquals("foo.com:5000", imageName.getRegistry());
-        assertEquals("customer-service-cache", imageName.getRepository());
-        assertEquals("latest", imageName.getTag());
+        assertThat(imageName).isNotNull();
+        assertThat(imageName.getUser()).isNull();
+        assertThat(imageName.getRegistry()).isEqualTo("foo.com:5000");
+        assertThat(imageName.getRepository()).isEqualTo("customer-service-cache");
+        assertThat(imageName.getTag()).isEqualTo("latest");
+    }
+
+    // TODO: Meaningful display name
+    @ParameterizedTest
+    @MethodSource("registryNamingTestData")
+    void testRegistryNaming(String imageName, String registry, String expected) {
+        assertThat(new ImageName(imageName).getFullName(registry)).isEqualTo(expected);
+    }
+
+    // TODO: Meaningful display name
+    @ParameterizedTest
+    @MethodSource("registryNamingExtendedTestData")
+    void testRegistryNamingExtended(String imageName, String registry, String expected) {
```

Comment:
https://github.com/eclipse/jkube/pull/1659/files#r921838135

Reply:
I'm slightly confused. This test is same as `testRegistryNamingExtended`. Why have you created it? Can't we test all these in `testRegistryNaming` ?

Reply:
Yes we can test all these in `testRegistryNaming` test, but I created it because it was there
https://github.com/eclipse/jkube/blob/c40de91d9604ee433f2403e87057078e4775644b/jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/ImageNameTest.java#L132

Reply:
I see. I think it's safe to merge both tests.